target/linux/ramips/image/mt7621.mk

```diff
@@ -231,7 +231,7 @@ TARGET_DEVICES += iodata_wn-gx300gr
 define Device/k2p
```

Comment:
[As in](https://github.com/openwrt/openwrt/pull/2138#discussion_r299796314) I'm wondering how the potential user is going to find out, that image for k2p also supports ke2p device as the firmware image naming is unchanged, thus there will be only k2p in the firmware filename.

Reply:
IMO, if one really wants to achieve that, one would have to add a device definition for ke2p (just copied from k2p) and duplicate lines elsewhere (01_leds, 02_network, ...) as well as DTS.
This will cause another image having to be built, but still I consider it the tidiest solution.
(I personally dislike merging both names into the device ID, e.g. phicomm_k2p-ke2p).

Nevertheless, this is a topic coming up regularly. I have a similar problem with the Ubiquiti XM devices (Bullet M, Loco M, Picostation M, ...): ubnt_bullet-m image is used for all of them, but I'd like to distinguish as all of them have different antenna_gain I need to set.